bestiary/core/models.py

```diff
@@ -68,6 +71,33 @@ def pre_save(self, model_instance, add):
         return value
 
 
+class EncryptedField(CharField):
+    """Field to store encrypted data using SECRET_KEY.
+
+    The max length for this field is 79 chars which
+    are 184 encrypted chars using Fernet encryption
```

Comment:
I think we should change this to AES 256. Although Fermet looks like a good choice I'd rather use the other one because many goberment agencies require that method.